schemas/public/ekscluster-kfd-v1alpha2.json

```diff
@@ -500,6 +500,9 @@
                 },
                 "vpcId": {
                     "$ref": "#/$defs/Types.AwsVpcId"
+                },
+                "bucketNamePrefix": {
+                    "$ref": "#/$defs/Types.AwsS3BucketName"
```

Comment:
You should create a new API version because the v1alpha2 has already been released.